### PR TITLE
Order details edit

### PR DESCRIPTION
**Pull request by mrrafi5869**

### CODE REVIEW

File: src/pages/order/edit-order/EditOrder.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    onError: (error) => {
+      console.log({ error });
+    },
+  });
+
+  useEffect(() => {
+    if (id) {
+      getAnOrder();
+    }
+  }, [getAnOrder, id]);
```
Comment (kamruz-zzaman) [inline src/pages/order/edit-order/EditOrder.jsx:87]: Id ace deilahi ei page a asce. If condition er dorkar nai

File: src/pages/order/edit-order/EditOrder.jsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    variables: { queryData: { id: id } },
+    errorPolicy: "all",
+    fetchPolicy: "no-cache",
+    onCompleted: (response) => {
+      setEditOrderData(response.getAnOrder);
+    },
+    onError: (error) => {
+      console.log({ error });
+    },
+  });
```
Comment (kamruz-zzaman) [inline src/pages/order/edit-order/EditOrder.jsx:81]: onError: console.log,

**Review by kamruz-zzaman — CHANGES_REQUESTED** [review]

File: src/pages/order/order-details/OrderDetails.jsx [hunk 3]
```diff
@@ -78,29 +78,41 @@ function OrderDetails() {
...
-              <p>Date of Birth: </p>
+            {data?.patientData?.dispatcher && (
+              <div className="flex items-center gap-6">
+                <p>Dispatcher: </p>
+                <p>{data?.patientData?.dispatcher}</p>
+              </div>
+            )}
+            <div className="flex items-center gap-6">
+              <p>Pick up: </p>
               <p>
```
Comment (mahmudulhaquequdrati) [inline src/pages/order/order-details/OrderDetails.jsx:93]: in pickup address why we have new Date()?

File: src/pages/order/edit-order/EditOrder.jsx (hunk 1 above)
Comment (mahmudulhaquequdrati) [reply src/pages/order/edit-order/EditOrder.jsx:87]: @mrrafi5869  solve it too

**Pull request merged**